tests/cache_test_state.rs

```diff
@@ -0,0 +1,81 @@
+use std::env;
+use std::sync::Arc;
+use std::time::Duration;
+
+use once_cell::sync::Lazy;
```

Comment:
did you guys talk about the different crates that are available for this, and just curious if there was a reason this one seemed best?

Reply:
if it's in std then why do we need the dep on once_cell? is the std thing only in nightly or something?

Reply:
discussed in slack - confirmed that the thing we need here is only in nightly, so we still need the crate